package.json

```diff
@@ -13,17 +13,18 @@
     "clean:node_modules": "npm run clean && rm -r ./node_modules 2> /dev/null || echo > /dev/null",
 
     "webpack": "webpack --progress --profile --colors --display-error-details --display-cached",
-    "prebuild": "tsfmt -r && mkdir build > /dev/null || echo > /dev/null && mkdir dist > /dev/null || echo > /dev/null && npm install",
-    "build": "npm run prebuild && npm run webpack",
+    "format": "tsfmt -r",
+    "prebuild": "mkdir build > /dev/null || echo > /dev/null && mkdir dist > /dev/null || echo > /dev/null && npm install",
+    "build": "npm run prebuild && npm run format && npm run webpack",
     "build:watch": "npm run prebuild && npm run webpack -- --watch",
 
     "start": "electron build",
     "start:build": "npm run build && npm run start",
 
     "electron-packager": "electron-packager build sinap --out=dist --no-prune --arch=all --overwrite",
     "prepackage": "npm run clean && npm run build && cp package.json build",
-    "package":"npm run prepackage && npm run electron-packager -- --platform=all",
```

Comment:
I'll be more careful when I edit it to be consistent. But you should really start using the one true code editor AKA VS Code ;)